aries-site/src/pages/whats-new.mdx

```diff
@@ -70,6 +72,7 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 - Added guiance and examples for [FileInput](/components/fileinput)
 - Added `kind="toolbar"` to button in grommet-theme-hpe for use in Filter contexts. See how this button is used in [Filtering examples](/templates/filtering).
+- Replaced Anchor Button with Defualt Button on when clearing filters on [Filtering examples on Design System site](https://design-system.hpe.design/templates/filtering) and [Figma](https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=1%3A3738).
```

Comment:
```suggestion
- Updated filtering pattern to use default Button instead of Anchor for "Clear Filters" control. View the [Filtering examples on Design System site](https://design-system.hpe.design/templates/filtering) and [Figma](https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=1%3A3738).
```